index.html

```diff
@@ -4,12 +4,219 @@
+              <li><a href="#">Events</a></li>
+              <li><a href="#">Business</a></li>
+              <li><a href="#">AMAs</a></li>
+              <li><a href="#">Spaces</a></li>
+              <li><a href="#">Terms & Conditions</a></li>
+            </ul>
+          
+            <ul class="nav-about flex">
+              <li><a href="#">About</a></li>
+              <li><a href="#">Advertise</a></li>
+              <li><a href="#">Jobs</a></li>
+              <li><a href="#">Contact</a></li>
+            </ul>
+          </div>
+
+          <div class="white-bar flex">
+            <ul class="white-bar-text flex">
+              <li><a href="#">LATEST</a></li>
+              <li><a href="#">HARD WORK</a></li>
+              <li><a href="#">PLUGGED</a></li>
+              <li><a href="#">README</a></li>
+              <li><a href="#">GROTH QUARTERS</a></li>
+          
+              <li><a class="white-bar-sec-text" href="#">SHIFT</a></li>
+              <li><a class="white-bar-sec-text" href="#">NEURAL</a></li>
+            </ul>
+
+            <ul class="nav-icons flex float-right">
+              <li><a href="#" class="search">
+                <img src="./images/computer-icons-pittsburg-button-search-box-button-ede7b4f4cdaaba07daadc3f70cc319f7.png" alt="">
+              </a></li>
+              <li><a href="#" class="menu-icon float-right">
+                <img src="./images/Hamburger_icon.svg.png" alt="">
+              </a></li>
+            </ul>
+          </div>
+
+          
+        </div>
+      </div>
+      <div class="yellow-bar flex">
+        <ul>
+          <li>Calling young digital talent in the Netherlands – Be recognized in the T500 →	</li>
+          
+        </ul>
+      </div>
+    </nav>
+
+    <main>
+      <div class="grid-one">
+        <div class="cover-image">
+            <a class="cover" href="#"><img src="images/tnw-logo.png" alt="logo" class="logo"></a>
+            <div class="header1">
+                <h1>Microsoft is adding AI-powered text predictions to Word</h1>
+                <p>by Thomas Macaulay </p>
+            </div>
+        </div>
+  
+        <div>
+          <div class="cover-image">
+            <div class="header1">
+              <h1>Microsoft is adding AI-powered text predictions to Word</h1>
+              <p>by Thomas Macaulay </p>
+            </div>     
+          </div>
+  
+          <div class="cover-image">
+            <div class="header1">
+              <h1>Microsoft is adding AI-powered text predictions to Word</h1>
+              <p>by Thomas Macaulay </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="funding">
+        <div class="funding-title">
+          <a href="#"><h2>Latest funding rounds on Index.co →</h2> </a>
+          <div class="funding-tags">
+            <a href="#"><h6>See more</h6></a>
+            <a href="#"><h6>Get updates by email</h6></a>
+            <a href="#"><h6>Add info</h6></a>
+          </div>
```

Comment:
This section should be a `nav` html semantic tag

Reply:
done